gallery/src/components/NxButton/NxButtonIconExample.tsx

```diff
@@ -5,16 +5,24 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import {faSync} from '@fortawesome/free-solid-svg-icons';
+import { faSync, faSave, faEdit } from '@fortawesome/free-solid-svg-icons';
 
-import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { NxButton, NxFontAwesomeIcon, NxTooltip } from '@sonatype/react-shared-components';
 
 const NxButtonIconExample = () =>
   <div className="nx-btn-bar">
-    <NxButton>
+    <NxButton aria-label="sync">
```

Comment:
Typically in a button containing both an icon and text, the icon is semantically redundant and should not have a label.

Reply:
> Typically in a button containing both an icon and text, the icon is semantically redundant and should not have a label.

https://github.com/sonatype/sonatype-react-shared-components/pull/247/commits/dedb255a7ff9c5c641fd7bfc853d02d008d45534